settings.gradle

```diff
@@ -15,3 +15,5 @@
 
 include 'core', 'examples', 'clients', 'tools', 'streams', 'streams:examples', 'log4j-appender',
         'connect:api', 'connect:transforms', 'connect:runtime', 'connect:json', 'connect:file', 'jmh-benchmarks'
+include 'log4j-controller'
```

Comment:
It may make sense not to do the module separation in the initial PR. It's not too bad to have the log4j JAR in the classpath for people using the tarball and one can use an exclusion if depending on the Kafka Core jar as a library. The separate module change would require a KIP if it has compatibility implications.